tests/console/curl_upload_logs.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
```

Comment:
better test module name could be "system_state.pm", no need to mention curl, which is a tool, not the purpose